### ISSUE TITLE
Telemetry: Collect the name of the installation profile

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5187]
This is part of the #285 meta.
Related: #3205

**Comment by klonos** [comment]
PR up for review:

![image](https://user-images.githubusercontent.com/2423362/131249745-27573f87-d04a-4026-b856-1a63446be168.png)

**Comment by stpaultim** [comment]
I guess, I don't object to collecting this data. However, given that Backdrop CMS doesn't even give users the option to install anything but the standard profile during a normal installation, I expect the results to be pretty predictable. 

I believe that the only install profiles we're going to get, other than standard are custom install profiles that might be named in revealing ways about the company or user that created them. 

In short, I don't think collecting the name of the install profile will reveal anything we know except how folks that have custom install profiles are naming them. 

I suppose it might be useful to know how many people are using anything other than the standard install profile, but the collecting the names of the custom alternatives doesn't seem useful.

**Comment by indigoxela** [comment]
> given that Backdrop CMS doesn't even give users the option to install anything but the standard profile during a normal installation, I expect the results to be pretty predictable.

People upgrading from Drupal might have all sorts of profiles. Although that also means, that they get PHP notices (unrelated here, issue #4799).
Sites installed as Backdrop will almost all the time use "standard".

(IMO maybe more relevant: webserver - Apache, Nginx, ...)

**Comment by klonos** [comment]
There's a related discussion in Zulip, from back in July this year: https://backdrop.zulipchat.com/#narrow/stream/218635-Backdrop/topic/Config.20defaults/near/247417421

> ...given that Backdrop CMS doesn't even give users the option to install anything but the standard profile during a normal installation, I expect the results to be pretty predictable.

That is true, but if people go through the trouble to ever enable an installation profile, and we see that number increasing, then we'll be able to easily make informed decisions with things like #1999.

**Comment by philsward** [comment]
I see it as a "placeholder" for future information. It may not be helpful now (it may never be helpful) but the foundation is there if we do something with it.

Worst case it's never used, someone can yank the code out of the module and delete the table that holds the data. I'm guessing that would take less resources and less time than trying to get it re-added at some point down the road.

It can always be renamed or re-purposed to something else like recipe usage.

**Comment by BWPanda** [comment]
I'm fine with this idea. Code looks good, but PR needs rebasing and conflicts to be resolved...

**Comment by BWPanda** [comment]
In @klonos' absence, I was able to update the PR from the GitHub UI.

**Comment by indigoxela** [comment]
I'm predicting we'll end up with `99% standard profile`. :wink: Similar to the pointless "Operating System" info.

**Comment by quicksketch** [comment]
I'm kind of on the fence about this one, which is why I haven't merged it in the last couple of releases. But I guess we want it to be there when we need it, in the event that some alternative distribution of Backdrop were to become popular. We might also request that [Silkscreen CMS](https://www.silkscreencms.org/) record itself as some other install profile(?)

**Comment by quicksketch** [comment]
I merged https://github.com/backdrop/backdrop/pull/3713 into 1.x and 1.22.x because it does no harm. Note that this won't start actually showing on https://backdropcms.org/project/backdrop/telemetry until we also configure the Telemetry settings on backdropcms.org. Perhaps for now we can actually not list it. We're sending the information now but not displaying it.

**Comment by klonos** [comment]
Noting that I've updated https://backdropcms.org/node/179/edit/telemetry to include the new metric for `install_profile` 😉
